doc/sphinx/source/quickstart/installation.rst

```diff
@@ -171,29 +204,41 @@ Note that the ESMValTool source code is contained in the ``esmvaltool-python``
 package, so this package will always be installed as a dependency if you install
 one or more of the packages for other languages.
 
-There is also a lesson available in the
-`ESMValTool tutorial <https://esmvalgroup.github.io/ESMValTool_Tutorial/>`_
-that describes the installation of the ESMValTool in more detail. It can be found
-`here <https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html>`_.
+.. _install_julia_dependencies:
+
+Installation of Julia dependencies
+----------------------------------
 
+If you want to use the ESMValTool Julia functionality, you will also need to
+install Julia. If you are just getting started, we suggest that you
+come back to this step later when and if you need it.
+To perform the Julia installation, make sure that your conda
+environment is activated and then execute
+
+.. code-block:: bash
+
+    mamba install julia
```

Comment:
why this? Julia is installed as a dep at the get-going

Reply:
No, it's commented out, see: https://github.com/conda-forge/esmvaltool-suite-feedstock/blob/main/recipe/meta.yaml

Reply:
ah of course it is! I am being silly. BTW I just tested the install procedure (the compressed `mamba create --name esmvaltool esmvaltool-python` way and all is fine :partying_face: